product-input-fields-for-woocommerce.php

```diff
@@ -244,10 +244,9 @@ public static function ts_admin_notices_scripts() {
 		 * Add tracker completed.
 		 */
 		public static function init_tracker_completed() {
-			if ( isset( $_SERVER['HTTP_REFERER'] ) ) {
-				header( 'Location: ' . sanitize_text_field( wp_unslash( $_SERVER['HTTP_REFERER'] ) ) );
-				exit;
-			}
+			$redirect_url = isset( $_SERVER['HTTP_REFERER'] ) ? sanitize_text_field( wp_unslash( $_SERVER['HTTP_REFERER'] ) ) : admin_url( 'admin.php?page=wc-settings&tab=alg_wc_pif' );
```

Comment:
@PratapDungrani I think it should be `sanitize_url` and not `sanitize_text_field` since we're dealing with a URL in this case.